src/FAQ/RestAPI/FAQField.php

```diff
@@ -12,35 +10,35 @@
  */
 class FAQField extends CreatesFields
 {
-
     /**
      * Create an additional field on an array.
-     *
-     * @param WP_Post $post
-     *
-     * @return array
      */
     public function create(WP_Post $post): array
     {
         return array_map(function ($faq) {
             return [
                 'question' => $faq['pdc_faq_question'] ?: '',
-                'answer'   => isset($faq['pdc_faq_answer']) ? apply_filters('the_content', $faq['pdc_faq_answer']) : ''
+                'answer'   => ! empty($faq['pdc_faq_answer']) ? apply_filters('the_content', $this->alterYouTubeEmbedURL($faq['pdc_faq_answer'])) : ''
             ];
         }, $this->getFAQ($post));
     }
 
     /**
      * Get faqs of a post.
-     *
-     * @param WP_Post $post
-     *
-     * @return array
      */
-    private function getFAQ(WP_Post $post)
+    private function getFAQ(WP_Post $post): array
     {
         return array_filter(get_post_meta($post->ID, '_owc_pdc_faq_group', true) ?: [], function ($faq) {
             return ! empty($faq['pdc_faq_question']) && ! empty($faq['pdc_faq_answer']);
         });
     }
+
+    protected function alterYouTubeEmbedURL(string $content): string
+    {
```

Comment:
Personally not a huge fan of comments next to code, maybe create a oneliner just above these 3 rules or function?